server/views/admin/users/edit.njk

```diff
@@ -4,83 +4,90 @@
 {% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
-{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% set pageTitle = applicationName + " - " + pageHeading %}
 {% set mainClasses = "app-container govuk-body assessments--index" %}
 
 {% block beforeContent %}
     {{ govukBackLink({
-		text: "Back",
-		href: paths.admin.userManagement.index()
-	}) }}
+        text: "Back",
+        href: paths.admin.userManagement.index()
+    }) }}
 {% endblock %}
 
 {% block content %}
-
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             {% include "../../_messages.njk" %}
 
             <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+
             <form action="{{ paths.admin.userManagement.update({id: user.id}) }}?_method=PUT" method="post">
                 <input type="hidden" name="_csrf" value="{{ csrfToken }}">
 
-                {{
-                govukSummaryList({
+                {{ govukSummaryList({
                     rows: UserUtils.userSummaryListItems(user)
-                })
-            }}
+                }) }}
 
                 {{ govukCheckboxes({
-                name: "roles",
-                fieldset: {
-                    legend: {
-                    text: "Select role access",
-                    classes: "govuk-fieldset__legend--m"
-                    }
-                },
-                hint: {
-                    text: "Select all that apply"
-                },
-                items: UserUtils.rolesToCheckboxItems(roles, user.roles)
-            }) }}
+                    name: "roles",
+                    fieldset: {
+                        legend: {
+                            text: "Select role access",
+                            classes: "govuk-fieldset__legend--m"
+                        }
+                    },
+                    hint: {
+                        text: "Select all that apply"
+                    },
+                    items: UserUtils.rolesToCheckboxItems(roles, user.roles)
+                }) }}
+
+                {% set allocationPreferencesHint %}
```

Comment:
Nice to move this to a variable, but how about moving the declarations to the top of the file so they don't interrupt the flow.